src/components/BackToTop.vue

```diff
@@ -35,5 +44,12 @@ function onClick() {
 .fab {
   margin: auto 8px 64px 8px;
   z-index: 1;
+  right: 23px;
+  bottom: 0;
+}
+@media (max-width: 600px) {
+  .fab {
+    right: 13px;
```

Comment:
Pref multiples of 4 to keep the grid easier to manage. Either 12px or 16 please, at your discretion